stories/SQFormDialogStepper.stories.js

```diff
@@ -273,6 +286,16 @@ export const SQDialogStepperWithValidation = () => {
             age: Yup.string().required('Required')
           }}
         >
+          <SQFormResetInitialValuesButton
+            tooltip="Reset Account Details"
+            confirmationContent="You are about to reset this form. Any unsaved info for this customer will be removed"
+            initialValuesObject={{
```

Comment:
same comment down here about creating one object for the initial values of this step and passing it here and to the initialValues above.